src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -155,6 +156,20 @@ void BLEManagerImpl::DriveBLEState()
         if (ConfigurationMgr().IsFullyProvisioned())
         {
             mFlags.Clear(Flags::kAdvertisingEnabled);
+
+            if (mFlags.Has(Flags::kChipoBleGattServiceRegister))
```

Comment:
Is it needed any more? `StopAdvertising` should take care of unregistering it now, no?